src/index.html

```diff
@@ -21,7 +24,372 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="#"
+      >
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--active is-active"
```

Comment:
You're already using `nav__link--active`. It's not good to use the `is-active` class together, as it can create redundancy in your code.